aws/accountid_endpoint_mode.go

```diff
@@ -0,0 +1,12 @@
+package aws
+
+// AccountIDEndpointMode switches on/off the account ID based endpoint routing
+type AccountIDEndpointMode string
+
+// enums of valid AccountIDEndpointMode
+const (
+	AccountIDEndpointModeUnset     AccountIDEndpointMode = ""
+	AccountIDEndpointModePreferred AccountIDEndpointMode = "preferred"
```

Comment:
[nonblocking] You can leave the type declaration off of everything but the first line, the compiler figures it out

e.g.
```go
type Foo = string

const (
    Foo1 Foo = "1"
    Foo2     = "2" // is implicitly type Foo as is everything after it
)